src/local-editor-service.ts

```diff
@@ -164,84 +185,95 @@ export class LocalEditorService extends EditorService {
           this.getAncestorDomModuleForElement(document, location.element);
       if (!domModule || !domModule.id)
         return;
-      const outerElement = document.getOnlyAtId(
-          'element', domModule.id, {imported: true, externalPackages: true});
+      const [outerElement] = document.getFeatures({
+        kind: 'element',
+        id: domModule.id,
+        imported: true,
+        externalPackages: true
+      });
       if (!outerElement)
         return;
       const sortPrefixes = this._createSortPrefixes(outerElement);
-      const innerElement = document.getOnlyAtId(
-          'element', location.element.nodeName,
-          {imported: true, externalPackages: true});
+      const [innerElement] = document.getFeatures({
+        kind: 'element',
+        id: location.element.nodeName,
+        imported: true,
+        externalPackages: true
+      });
       if (!innerElement)
         return;
-      const innerAttribute = innerElement.attributes.find(
-          (value) => value.name === location.attribute);
+      const innerAttribute = innerElement.attributes.get(location.attribute);
       if (!innerAttribute)
         return;
       const attributeValue =
           dom5.getAttribute(location.element, innerAttribute.name)!;
       const hasDelimeters = /^\s*(\{\{|\[\[)/.test(attributeValue);
       return {
         kind: 'attribute-values',
-        attributes: outerElement.properties.map(p => {
+        attributes: [...outerElement.properties.values()]
+                        .map(p => {
```

Comment:
Indentation is rough here. Since the map function is quite long, maybe extract it already?